src/core/url_store/atoms/urlStore.ts

```diff
@@ -111,9 +112,9 @@ export const urlStoreAtom = createAtom(
     const currentMapPosition = get('currentMapPositionAtom');
     if (currentMapPosition) {
       newState.map = [
-        Number(currentMapPosition.zoom.toFixed(3)),
-        Number(currentMapPosition.lng.toFixed(3)),
+        Number(currentMapPosition.zoom.toFixed(3)) + URL_ZOOM_OFFSET,
         Number(currentMapPosition.lat.toFixed(3)),
+        Number(currentMapPosition.lng.toFixed(3)),
```

Comment:
add backlog note - the toFixed(3) will break link sharing on smart city zoom levels.